source/deploy/mobile-app-changelog.md

```diff
@@ -1,12 +1,49 @@
 # Mattermost mobile apps changelog
 
 Latest Mattermost Mobile Apps releases:
-- [1.50.1 Release](#release)
-- [1.50.0 Release](#id1)
-- [1.49.1 Release](#id5)
-- [1.49.0 Release](#id8)
-- [1.48.2 Release](#id14)
-- [1.48.1 Release](#id19)
+- [1.51.0 Release](#release)
+- [1.50.1 Release](#id1)
+- [1.50.0 Release](#id5)
+- [1.49.1 Release](#id11)
+- [1.49.0 Release](#id14)
+- [1.48.2 Release](#id20)
+
+## 1.51.0 Release
+- Release Date: April 16, 2022
+- Server Versions Supported: Server v6.3.0+ is required. Self-Signed SSL Certificates are not supported unless the user installs the CA certificate on their device.
+
+### Compatibility
+ - **Upgrade to server version v6.3.0 or later is required.** Support for server [Extended Support Release](https://docs.mattermost.com/upgrade/extended-support-release.html) (ESR) 5.37 has ended and upgrading to server ESR v6.3.0 or later is required. As we innovate and offer newer versions of our mobile apps, we maintain backwards compatibility only with supported server versions. Users who upgrade to the newest mobile apps while being connected to an unsupported server version can be exposed to compatibility issues, which can cause crashes or severe bugs that break core functionality of the app.
+ - Android operating system 7+ [is required by Google](https://android-developers.googleblog.com/2017/12/improving-app-security-and-performance.html).	
+ - iPhone 5s devices and later with iOS 12.1+ is required.
+
+### Breaking Changes
+ - Apps calls are now separated between submit, form, refresh and lookup calls.
+
+### Improvements
+ - Users are now taken directly to the SSO login if only one SSO login method is enabled.
+ - Calls are now enabled on mobile if the server has Calls enabled.
```

Comment:
Do we want to mention Calls already in the changelog, or do we remove the Calls items for now?

Reply:
@itao ☝️

Reply:
I checked -- we're okay to release Calls-related changelog entries.